README.md

```diff
@@ -59,34 +86,34 @@ And ask someone for the values to set the following environment variables:
 _You can also `export` the above environment variables to your `.bash_profile`,
 or use your favorite alternative method of setting environment variables._
 
-__NOTE__ about GovDelivery: GovDelivery is a third party web service that powers our subscription forms. Users may decide to swap this tool out for another third party service. The application will function but throw an error if the above GovDelivery values are not set. 
-
-### Front end setup
+**NOTE:** GovDelivery is a third-party web service that powers our subscription forms.
+Users may decide to swap this tool out for another third-party service.
+The application will function but throw an error if the above GovDelivery values are not set.
 
-The cfgov-refresh front end currently uses the following frameworks / tools:
+## Usage
 
-- [Grunt](http://gruntjs.com/)
-- [Bower](http://bower.io/)
-- [Less](http://lesscss.org/)
-- [Capital Framework](http://cfpb.github.io/capital-framework/getting-started/)
+Generally you will have four tabs (or windows) open in your terminal when using this project.
+These will be used for:
+ 1. **Git operations**. Perform git operations and general development in the repository.
+ 2. **Elasticsearch**. Run Elasticsearch instance.
+ 3. **Sheer web server**. Run the web server.
+ 4. **Grunt watch**. Run Grunt watch task for watching for changes to content.
 
+What follows are the specific steps for each of these tabs.
 
-If you're new to Capital Framework, we encourage you to
-[start here](http://cfpb.github.io/capital-framework/getting-started/).
+### 1. Clone project and pull in latest updates
 
-#### Installing dependencies (one time)
+Using Terminal, navigate to your project directory (`cd ~/Projects` or equivalent).
+Clone the project with:
 
-1. Install [node.js](http://nodejs.org/) however you'd like.
-2. Install [Grunt](http://gruntjs.com/) and [Bower](http://bower.io/):
-3. 
-```
-$ npm install -g grunt-cli bower
+```bash
+$ git clone git@github.com:cfpb/cfgov-refresh.git
+$ cd cfgov-refresh
 ```
 
-### Developing
-
-Each time you fetch from upstream, you should install dependencies with npm and
-`grunt vendor`, then run `grunt` to rebuild everything:
+Each time you fetch from the upstream repository (this repo),
+you should install dependencies with npm and `grunt vendor`,
```

Comment:
Should we include `update` to installing NPM dependencies? The `install` command won't update existing dependencies. "NPM" should also be uppercase here because it's the name and not the command.